### DISCUSSION TITLE
Scroll not working when placing custom layout in a Scroll view.

### DISCUSSION THREAD

**Discussion by MohamedHasan3644** [dotnet/maui#21315] [Q&A]
### Description

I have created the custom layout and added it inside the scroll view while tapped and panning outside the custom layout the scrolling occurs, but while tapped and panning inside the custom layout scrolling does not occurs.

Below I have attached the code snippet to reproduce the issue.
```C#
 var scrollView = new ScrollView();
var stackLayout = new StackLayout();
 
// Adding labels outside the custom grid
stackLayout.Children.Add(new Label { Text = "Hello", HeightRequest = 500 });
 
// Adding custom grid
var customGrid = new Grid { BackgroundColor = Colors.Red };
for (int i = 0; i < 4; i++)
{
    customGrid.RowDefinitions.Add(new RowDefinition { Height = 100 });
    Label label = new Label() { Text = "Hello", HeightRequest = 100 };
    customGrid.SetRow(label, i);
    customGrid.Children.Add(label);
}

PanGestureRecognizer pan = new PanGestureRecognizer();
pan.PanUpdated += Pan_PanUpdated;
customGrid.GestureRecognizers.Add(pan);
stackLayout.Children.Add(customGrid);
 
// Adding labels outside the custom grid
stackLayout.Children.Add(new Label { Text = "Hello", HeightRequest = 500 });
scrollView.Content = stackLayout;
Content = scrollView; `
```

This issue is only on Windows, Mac and IOS

### Output:

https://github.com/dotnet/maui/assets/102596726/4adce40f-1ff0-43fa-9a3b-14eb1d54229c

### Steps to Reproduce

1. Create a .NET MAUI Project with the above code.
2. Run the project in windows, Mac and IOS.

### Link to public reproduction project repository

[ScrollingCheck7.zip](https://github.com/dotnet/maui/files/14636395/ScrollingCheck7.zip)

### Version with bug

Unknown/Other

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Windows, macOS

### Affected platform versions

IOS 16.2, Windows 10.0.19045

**Comment by PureWeen** [comment]
@MohamedHasan3644 I don't quite follow your use case here. 

If you're adding a PanGestureRecognizer that would seem to undo scrolling.

**Comment by MohamedHasan3644** [comment]
Hi @PureWeen is there any ways to achieve the scroll of the scroll view while pan gesture recognizer is added to the custom layout.

**Comment by dotnet-policy-service** [comment]
Hi @MohamedHasan3644. We have added the "s/try-latest-version" label to this issue, which indicates that we'd like you to try and reproduce this issue on the latest available public version. This can happen because we think that this issue was fixed in a version that has just been released, or the information provided by you indicates that you might be working with an older version.

You can install the latest version by installing the latest Visual Studio (Preview) with the .NET MAUI workload installed. If the issue still persists, please let us know with any additional details and ideally a [reproduction project](https://github.com/dotnet/maui/blob/main/.github/repro.md) provided through a GitHub repository.

This issue will be closed automatically in 7 days if we do not hear back from you by then - please feel free to re-open it if you come back to this issue after that time.

**Comment by ninachen03** [comment]
Verified on VS for Mac 17.6.10 build 428 , can not repro issue with sample project on MacCatalyst platform &iOS platform.and also works fine on 17.10.0 Preview 2.0 at windows platform.

![Scroll](https://github.com/dotnet/maui/assets/128687574/933cecb9-d177-4708-8ca1-b361d5fdb0d7)

**Comment by PureWeen** [comment]
> Hi @PureWeen is there any ways to achieve the scroll of the scroll view while pan gesture recognizer is added to the custom layout.

I'm still not clear on your overall use case. 
Scrolling and panning seem like contradictory interactions.

**Reply by MuniappanSubramanian** [reply]
Hi @PureWeen 

We have added a pan gesture to the custom grid, which is a child of the scroll view. When I tap on the grid and try to scroll, the parent scroll view does not scroll. However, if I tap on a label that is added as a child, the parent scroll view works. The scroll view should also scroll when tapping and dragging on the custom grid.

**Reply by munkii** [reply]
@PureWeen 

I just tried this on a dev machine using Microsoft.Maui.Controls 9.0.51 and a` dotnet workload list ` of

| Installed Workload Id  | Manifest Version | Installation Source |
| ------------- | ------------- | ------------- |
| android   | 35.0.61/9.0.100  | SDK 9.0.300, VS 17.14.36109.1  |
| ios  | 18.4.9288/9.0.100  | SDK 9.0.300, VS 17.14.36109.1  |
| maccatalyst  | 18.4.9288/9.0.100  | SDK 9.0.300, VS 17.14.36109.1  |
| maui-windows| 9.0.51/9.0.100  | SDK 9.0.300, VS 17.14.36109.1  |

It is very much still an issue.

> I'm still not clear on your overall use case.

We have a use case where this issue is blocking us. Imagine a calendar control that displays a month at a time. You want to enable navigation within that control from one month to the next. Whilst you could do that with buttons you also want to enable horizontal swiping.

Now if you have two dates, like a start date and an end date (See Add appointment in iOS native calendar app) you quickly have a need to be able to scroll the page vertically except you cannot.

Imagine the Red panel in the sample code from @MohamedHasan3644 being that Calendar.

**Reply by munkii** [reply]
@PureWeen do you still think there is no use case?

**Comment by VishalOmprasad** [comment]
Hi team,

Following up - are there any updates or progress on this issue?

**Comment by HadayaRahman-SF4673** [comment]
Hi Team,
Is there any update on this issue ?
